src/redux/actions/experimentSettings/updateSampleFilterSettings.js

```diff
@@ -0,0 +1,10 @@
+import { EXPERIMENT_SETTINGS_SAMPLE_FILTER_UPDATE } from '../../actionTypes/experimentSettings';
+
+const updateSampleFilterSettings = (step, sampleId, diff) => (dispatch) => {
```

Comment:
Didnt updateProcessingSettings take care of this or?

Reply:
Yes, I guess actually some thing I could do is change `updateProcessingSettings` to be blocked to only handling the steps that aren't separated samples and see if I have to add more separation there.

Reply:
And Ill also add comments.

Reply:
Ended up just changing the name, if you can think of a better one please let me know. I want it to say that it is only for configure embedding, data integration or meta, basically all the updates that are for something that doesn't work on a per-sample basis.